drools-eclipse/pom.xml

```diff
@@ -117,7 +117,7 @@
           <artifact>
             <groupId>org.jboss.tools.integration-stack</groupId>
             <artifactId>target-platform</artifactId>
-            <version>4.3.0.Beta1f-SNAPSHOT</version>
+            <version>4.3.1-SNAPSHOT</version>
```

Comment:
Bob, is there a need to use a SNAPSHOT here, what about staying on the latest stable release instead?  While in theory we can live with SNAPSHOTS in between releases, I'm a little afraid we'd forget about it and only discover potential issues too late when we try to do a release.